Directory.Build.props

```diff
@@ -13,6 +13,8 @@
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
         <NoWarn>NU5105;NU5128;NU5100;NU5118;0419,0649</NoWarn>
+        <EnableNETAnalyzers>true</EnableNETAnalyzers>
+        <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
I think they will also enable the analyzers in source build. I'm not familiar with source build. Hopefully it will not affect source build.

Reply:
Afaik, source build has a problem with including analyzer packages, not actually running the checks.
I feel this is a better place for properties, together with `TreatWarningsAsErrors` and `NoWarn`.
`Analysers.props` was used to include additional analyzers.